tlsutil/config.go

```diff
@@ -164,9 +197,9 @@ func (c *Config) OutgoingTLSWrapper() (DCWrapper, error) {
 	// Generate the wrapper based on hostname verification
 	if c.VerifyServerHostname {
 		wrapper := func(dc string, conn net.Conn) (net.Conn, error) {
-			conf := *tlsConfig
+			conf := clone(tlsConfig)
```

Comment:
@slackpad You might want to add a comment that in go1.8 `tls.Config.Clone()` will be exported. 

https://github.com/golang/go/commit/d24f446